src/main.js

```diff
@@ -75,18 +71,18 @@ function escape(
  * @param {string} args.arg The argument to escape.
  * @param {string} args.shellName The name of the shell to escape `arg` for.
  * @param {object} deps The dependencies for this function.
- * @param {Function} deps.getEscapeFunction Get the escape function for a shell.
  * @param {Function} deps.getQuoteFunction Get the quote function for a shell.
  * @returns {string} The quoted and escaped argument.
  * @throws {TypeError} The argument to escape is not stringable.
  */
-function quote({ arg, shellName }, { getEscapeFunction, getQuoteFunction }) {
-  const escapedArg = escape(
-    { arg, interpolation: false, quoted: true, shellName },
-    { getEscapeFunction }
-  );
+function quote({ arg, shellName }, { getQuoteFunction }) {
+  if (!isStringable(arg)) {
+    throw new TypeError(typeError);
+  }
+
+  const argAsString = arg.toString();
```

Comment:
Can duplication between this :arrow_down: be avoid?:

https://github.com/ericcornelissen/shescape/blob/6d4b4cbb9152754d9e3fa84d722af6743f4bcd93/src/main.js#L56-L61